tools/cve/tag_map.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright 2022 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -Eeo pipefail
+shopt -s failglob
+
```

Comment:
There is no description what is the purpose of this script. What data it takes and what it returns.

Reply:
I've added a description for tag_map.sh in bf15eeb417a4481e70bd9e47406737751e8d2f61.

Reply:
I've added a description for tag_map.sh in bf15eeb417a4481e70bd9e47406737751e8d2f61.